pkgs/applications/misc/ArchiSteamFarm/default.nix

```diff
@@ -1,42 +1,51 @@
-{ lib, stdenv, fetchurl, unzip, makeWrapper, dotnetCorePackages, jq }:
-
-stdenv.mkDerivation rec {
-  pname = "ArchiSteamFarm";
-  version = "4.3.1.0";
-
-  src = fetchurl {
-    url = "https://github.com/JustArchiNET/ArchiSteamFarm/releases/download/${version}/ASF-generic.zip";
-    sha256 = "1q28byshh4wkfsfdb0sfdqq9a5da9k7i4nagsfpk0fzyajvzd4lx";
+{ lib
+, buildDotnetModule
+, fetchFromGitHub
+, dotnetCorePackages
+, libkrb5
+, zlib
+, openssl
+}:
+
+buildDotnetModule rec {
+  pname = "archisteamfarm";
+  version = "5.1.5.3";
+
+  src = fetchFromGitHub {
+    owner = "justarchinet";
+    repo = pname;
+    rev = version;
+    sha256 = "0y69yazayrww5xywkbcgqjspz95hy73qnnx8dkkfhrl7laczqk8z";
```

Comment:
I ran the update script just to test, and it gave me a different hash: `sha256-H038maKHZujmbKhbi8fxsKR/tcSPrcl9L5xnr77yyXg=`. After replacing it with lib.fakeSha256 and then running again the error also included my hash, and not the one you committed here.